app.yaml

```diff
@@ -3,10 +3,56 @@ modules:
     config:
       "@type": cosmos.app.runtime.v1alpha1.Module
       app_name: Noble
-      pre_blockers: [ upgrade ]
-      begin_blockers: [ capability, authority, slashing, evidence, staking, ibc, authz, swap, ratelimit, dollar ]
-      end_blockers: [ crisis, staking, feegrant, forwarding, ratelimit ]
-      init_genesis: [ capability, transfer, auth, bank, staking, slashing, crisis, fiat-tokenfactory, globalfee, genutil, ibc, interchainaccounts, packetfowardmiddleware, evidence, authz, feegrant, params, upgrade, vesting, cctp, forwarding, aura, halo, florin, authority, wormhole, dollar, swap, hyperlane, warp, ratelimit ]
+      pre_blockers: [upgrade]
+      begin_blockers:
+        [
+          capability,
+          authority,
+          slashing,
+          evidence,
+          staking,
+          ibc,
+          authz,
+          swap,
+          ratelimit,
+          dollar,
+        ]
+      end_blockers: [crisis, staking, feegrant, forwarding, ratelimit]
+      init_genesis:
+        [
+          capability,
+          transfer,
+          auth,
+          bank,
+          staking,
+          slashing,
+          crisis,
+          fiat-tokenfactory,
+          globalfee,
+          genutil,
+          ibc,
+          interchainaccounts,
+          packetfowardmiddleware,
+          evidence,
+          authz,
+          feegrant,
+          params,
+          upgrade,
+          vesting,
+          cctp,
+          forwarding,
+          orbiter,
```

Comment:
Why did you add it here and not just append to the list like we've been doing for the last few major releases? 🤔